### ISSUE TITLE
China, PLAAF, WZ-8 fuel consumption in different altitude

### ISSUE THREAD

**Issue by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#4500]
### DB Selector

DB3K

### Affected DBID(s)

Aircraft 6642

### Description of Problem

1. According to general aerodynamic principles, the higher the atmospheric space, the lower the air drag, the lower the fuel consumption of the aircraft should be. However, the speed performance of the WZ-8 is contrary to common sense, the higher the altitude, the more fuel consumption. We all know that all kinds of missiles and aircraft have increased their altitude to achieve greater range and more economical fuel consumption. Hypersonic vehicles (similar to the wz-8) can achieve longer unpowered glide distances in close space.

![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/107970049/c225e206-161a-4df2-8eac-a8189b82b08c)

2. The description said, at its maximum cruising altitude, the WZ-8 has a reconnaissance radius of 900 nautical miles, while in the game, the WZ-8 at its maximum flying altitude (Hi-Hi-Hi) has a combat radius of less than 500 nautical miles. This is the ripple effect of the first problem above.

![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/107970049/630e84cf-01b3-4954-98d4-3a7dce16e0f6)

### Recommended Fix

 **Appropriate reduction of WZ-8 fuel consumption** at **higher altitudes** to enable it to achieve the desired combat radius at maximum altitude. That is: **combat radius of not less than 900 nautical miles/maximum range of not less than 2000 nautical miles (considering 10% fuel reserve)** at **50km altitude** and speed of not less than **Mach 3 /2000kt**.

**Comment by PygmalionOfCyprus** [comment]
I've made some adjustments that should make gameplay more viable with the WZ-8 (the cruise distance was set to 900nmi, not 2000nmi, which is why it's failing to achieve its recon radius) but the "higher SFC at high altitudes" is going to have to wait a little bit, since we don't actually have rocket propulsion for aircraft yet and it's currently using turbojets as a placeholder (which do struggle in thinner air). So, consider partially fixed, but leaving the ticket open for now.

**Comment by LuigiP57** [comment]
> I've made some adjustments that should make gameplay more viable with the WZ-8 (the cruise distance was set to 900nmi, not 2000nmi, which is why it's failing to achieve its recon radius) but the "higher SFC at high altitudes" is going to have to wait a little bit, since we don't actually have rocket propulsion for aircraft yet and it's currently using turbojets as a placeholder (which do struggle in thinner air). So, consider partially fixed, but leaving the ticket open for now.

Well, let's take D-21, which is about the same size.

![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/107970049/9157af53-e15a-426a-be47-dbaa15a9c7af)

The D-21 is thought to be comparable in size to the WZ-8, and the ramjet D-21 can fly 3,000 nautical miles without a disposable fuel tank. 

 If we don't insist on the same range as the D-21's 3,000 nautical miles, then 2,000 nautical miles is reasonable, right?

I sincerely hope you can reconsider it.

**Comment by PygmalionOfCyprus** [comment]
Yes, sorry for the misunderstanding: what I meant by my earlier comment was that the range had been initially set to 900nmi, but I have changed it to 2000nmi as part of the fix for the problems you've reported.

**Comment by PygmalionOfCyprus** [comment]
As of right now "rocket engine aircraft" is suitably obscure enough of a use case that we aren't planning on implementing the DB schema and sim-side implementation in the short-term. So, we'll have to stick with the imperfect turbojet simulation for now, which will mean some wonky fuel burn numbers at altitude but shouldn't affect gameplay too much. The important thing is that the range issue you reported should have been fixed back in...whatever DB update I fixed it in.

If WZ-8 is still not WAD lemme know, but for now I'm gonna close this.
